operator/controllers/seldondeployment_controller.go

```diff
@@ -889,8 +889,13 @@ func createDeploymentWithoutEngine(depName string, seldonId string, seldonPodSpe
 	if deploy.Spec.Template.Spec.SchedulerName == "" {
 		deploy.Spec.Template.Spec.SchedulerName = "default-scheduler"
 	}
+
+	// Set TerminationGracePeriodSeconds
 	var terminationGracePeriod int64 = 20
 	deploy.Spec.Template.Spec.TerminationGracePeriodSeconds = &terminationGracePeriod
+	if seldonPodSpec != nil && seldonPodSpec.Spec.TerminationGracePeriodSeconds != nil {
```

Comment:
Would zero be a vaild termination grace period or shoudl taht be ignored?

Reply:
I think its valid